### ISSUE TITLE
Intermittent time-out issue in MCP

### ISSUE THREAD

**Issue by slesaad** [US-GHG-Center/ghgc-stac-ingestor#10]
#PI Objective

# Description
The ingestor API is facing an intermittent time-our issue in MCP (staging and production).
We thought it might have been related to https://github.com/US-GHG-Center/ghgc-stac-ingestor/issues/6, but it was not.

Figure out why, and fix.

# Acceptance Criteria
- [x] The API no longer times out

**Comment by slesaad** [comment]
Following the tests done in #6 , I created a new lambda in MCP in the same VPC, subnets and security groups. This lambda did a simple `GET` request to `google.com`.

Even then the lambda was timing out, so it's not the source code that's the issue.

So looked at the NACL associated with the VPC, and in the inbound rules, in line 999, the port range of 32768-65535 is what's causing the issue.

Confirmed by MCP:
>I was out yesterday and am just getting to reviewing ticket responses. The issue is the NACL and it needs to be updated. Lambda requires the full ephemeral range which is 1024-65535. I'll put this through as a change.
https://docs.aws.amazon.com/vpc/latest/userguide/vpc-network-acls.html#nacl-ephemeral-ports
![](https://files.slack.com/files-pri/T3T8FQUK0-F05M1S2K5PC/image.png)

The NACL is being updated by MCP now.
